### PR TITLE
Dev jasper

### PR DESCRIPTION
**Pull request by Toast3347**
Login in finished and should work now.

### CODE REVIEW

File: DAL/LoginDAO.cs [hunk 1]
```diff
@@ -10,9 +12,14 @@ namespace DAL
 {
     public class LoginDAO : DAO
     {
-        public void Test()
+        public Employee GetUserByID(string username)
         {
-            List<Ticket> tickets = GetTicketCollection().Find(ticket => ticket.Status == ETicketStatus.Open).ToList();
+            Employee newEmployee = (Employee)GetEmployeeCollection().Find(employee => employee.FirstName == username).FirstOrDefault();
```
Comment (iolite312) [inline DAL/LoginDAO.cs]: The cast is not needed because to GetEmployeeCollection is already typed

File: Logic/LoginService.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logic
+{
+    public class LoginService
+    {
+        LoginDAO loginDAO;
+        EncryptionService encryptionService;
```
Comment (iolite312) [inline Logic/LoginService.cs]: Please put private before the variables if they are not needed outside of this class

**Review by iolite312 — CHANGES_REQUESTED** [review]
Added 2-3 comments if you will look at that, everything else is A OK

File: Logic/LoginService.cs (hunk 2 above)
Comment (Toast3347) [reply Logic/LoginService.cs]: Fixed!

File: DAL/LoginDAO.cs (hunk 1 above)
Comment (Toast3347) [reply DAL/LoginDAO.cs]: Fixed!

**Review by iolite312 — APPROVED** [review]
Good now it can be merged

**Pull request merged**